EasyPeasy/Context.swift

```diff
@@ -0,0 +1,115 @@
+// The MIT License (MIT) - Copyright (c) 2016 Carlos Vidal
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+#if os(iOS)
+    
+import UIKit
+
+/**
+     Struct that from an `UITraitCollection` object populates some helper 
+     properties easing access to device and size class information
+ */
+public struct Context {
+    
+    /// `true` if the current device is an iPad
+    public let isiPad: Bool
```

Comment:
Tricky, doing camel case with a noun that starts with a lowercase letter. I personally prefer `isPad` or `isIPad` but it's up to you!